Casks/a/agent-tars.rb

```diff
@@ -7,9 +7,22 @@
   desc "Multimodal AI agent for GUI interaction"
   homepage "https://github.com/bytedance/UI-TARS-desktop"
 
+  # recent releases include latest, pre-release
   livecheck do
     url :url
     regex(/^Agent[._-]TARS[._-]v?(\d+(?:\.\d+)+(?:[._-]alpha[._-]?\d+)?)$/i)
+    strategy :github_releases do |json, regex|
+      json.map do |release|
+        next if release["draft"]
+
+        release["assets"]&.map do |asset|
+          match = asset["name"]&.match(regex)
+          next if match.blank?
+
+          match[1]
+        end
+      end.flatten
```

Comment:
```suggestion
        # TODO: Reinstate pre-release skipping when the cask is updated to a
        # stable release.
        next if release["draft"] # || release["prerelease"]

        match = release["tag_name"]&.match(regex)
        next if match.blank?

        match[1]
      end
```

This check is failing because the existing regex is intended to match tags but this `strategy` block is matching against release asset filenames instead. This suggestion updates the strategy block to check the release tag name instead.